jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketMessageMetadata.java

```diff
@@ -58,14 +58,14 @@ public void setMaxMessageSize(int maxMessageSize)
         this.maxMessageSizeSet = true;
     }
 
-    public MethodHandle getMethodHandle()
+    public MethodHolder getMethodHolder()
     {
-        return methodHandle;
+        return methodHolder;
     }
 
-    public void setMethodHandle(MethodHandle methodHandle)
+    public void setMethodHolder(MethodHolder methodHolder)
```

Comment:
This setter and the getter have signature changes. Can this class be made internal?

Reply:
It is internal, anything in javax-websocket-core is internal.